pymongoexplain/commands.py

```diff
@@ -39,6 +39,8 @@ def command_name(self):
     def get_SON(self):
         cmd = SON([(self.command_name, self.collection)])
         cmd.update(self.command_document)
+        if self.command_document == {}:
+            return {}
```

Comment:
Why was this change needed?

Reply:
There were a few cases where the filter was {} and the expected command payload in that case was also {}

Reply:
It should not be possible for a command to be empty. Could you post those cases?

Reply:
It is in the following cases:
```
SON([('find', 'test_find_allowdiskuse'), ('filter', {})])
{}
```

Reply:
Which spec test is this specifically? Can you post the entire test failure output?